### DISCUSSION TITLE
hope show more items at once is available

### DISCUSSION THREAD

**Discussion by kokomaya** [damms005/devdb-vscode#78] [Q&A]
![image](https://github.com/user-attachments/assets/bd99a089-56dd-4321-bb70-a82616c521a5)
the most items could be shown at once is 50 only, if it possible to give more options, or user defined number ~

**Comment by damms005** [comment]
I think the available options are just right for the use case.

DevDb, by nature of being a VS Code extension, should be very lightweight and non-resource gulping. Setting an infinite upper bound to number of items that can be shown at once, I think, is not a good move because of this very point of the need to be lightweight.

However, for the sake of skipping ahead to items further down the list, I'll recommend using the pagination feature, as that allows you to jump to an arbitrary page.
